### PR TITLE
Issue-88 - Add proper readme for mapbox-events-ios

### PR DESCRIPTION
**Pull request by alfwatt**
https://github.com/mapbox/mapbox-events-ios/issues/88

- Add readme.md and Makefile to the Xcode project
- Remove MapboxMobileEvents.podspec from copy resources build phase
- Update Copyright dates in Info.plist
- Add (All Targets) aggregate scheme for quick compile testing

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,3 +2,43 @@
...
+
+    MMEventsManager* manager = [MMEventsManager.alloc 
+        initilizeWithAccessToken:@"your-mapbox-token" 
+        userAgentBase:@"user-agent-string"
+        hostSDKVersion:@"1.0.0"];
+    manager.delegate = self;
+    manager.isMetricsEnabledInSimulator = YES;
+    manager.isDebugLoggingEnabled = (DEBUG ? YES : NO);
+    manager.initilize(withAccessToken:@"your token kere")
+    [manager sendTurnstileEvents];
```
Comment (rclee) [inline readme.md]: nit: typo with `[manager sendTurnstileEvents];` -> [manager sendTurnstileEvent];

**Comment by alfwatt** [comment]
That force push was probably not 100% necessary, but I had to manually merge the pbxproj file

File: readme.md [hunk 2]
```diff
@@ -2,3 +2,43 @@
...
+    eventsManager.isMetricsEnabledInSimulator = true
+    eventsManager.isDebugLoggingEnabled = (DEBUG ? true : false)
+    eventsManager.initialize(withAccessToken: "your-mapbox-token", userAgentBase: "user-agent-string", hostSDKVersion: "1.0.0")
+    eventsManager.sendTurnstileEvent()
+
+The `userAgentBase` and `hostSDKVersion` strings are used to build the `UserAgent:` header for event reports.
+
+### Change Log
+
+- v0.8.1 — Fix for some events not reacing the server, and duplication when flushing
```
Comment (rclee) [inline readme.md]: nit: typo `reacing` -> `reaching`

File: readme.md [hunk 3]
```diff
@@ -2,3 +2,43 @@
 
 [![Bitrise](https://app.bitrise.io/app/63d52d847cdb36db/status.svg?token=DDdEMfpVR8emhdGSgToskA&branch=master)](https://www.bitrise.io/app/63d52d847cdb36db)
 ![codecov](https://codecov.io/gh/mapbox/mapbox-events-ios/branch/master/graph/badge.svg)
+
```
Comment (rclee) [inline readme.md:5]: We might include a short description of what the library does... Something similar to what we have on the [android repo](https://github.com/mapbox/mapbox-events-android#about-libtelemetry) or on our [telemetry page](https://www.mapbox.com/telemetry).

File: readme.md (hunk 1 above)
Comment (alfwatt) [reply readme.md]: good catch, the `.initilize(...)` call also completely wrong ;)

File: readme.md (hunk 2 above)
Comment (alfwatt) [reply readme.md]: Removing for now

File: readme.md [hunk 4]
```diff
@@ -2,3 +2,43 @@
...
+Or, in Swift:
+  
+    var eventsManager = MMEEventsManager()
+    eventsManager.isMetricsEnabledInSimulator = true
+    eventsManager.isDebugLoggingEnabled = (DEBUG ? true : false)
+    eventsManager.initialize(withAccessToken: "your-mapbox-token", userAgentBase: "user-agent-string", hostSDKVersion: "1.0.0")
+    eventsManager.sendTurnstileEvent()
+
+The `userAgentBase` and `hostSDKVersion` strings are used to build the `UserAgent:` header for event reports.
```
[the remark this answers is not in the document]
Comment (alfwatt) [reply readme.md]: Removing for now

File: readme.md (hunk 3 above)
Comment (alfwatt) [reply readme.md:5]: Included a brief description with a link to the telemetry page

**Review by rclee — APPROVED** [review]
LGTM 👍

**Pull request merged**